.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,25 @@
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "**" ]
+  pull_request:
+    branches: [ "**" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: maven
+    - name: run tests
+      run: mvn clean install
```

Comment:
why not "mvn clean verify"?

Reply:
Okay, just tested it and verify downloads dependencies. I wil change it to clean verify